analysis/scripts/sync-pairings.mjs

```
@@ -0,0 +1,237 @@
+  store.pairings ??= {};
+
+  const tally = { added: 0, merged: 0, unresolved: 0 };
+  const unresolved = [];
+
+  for (const set of [...SETS, ...NON_STANDARD_SETS]) {
+    let decks = [];
+    try {
+      decks = await fetchSet(set);
+    } catch (err) {
+      // One bad page must not discard the whole run's progress.
+      failures.push(`${set}: ${err.message}`);
```

ESLint

[error] 'failures' is not defined.

(no-undef)